app/javascript/stylesheets/about.scss

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=David+Libre&display=swap');
+
+.about-title {
+  font-weight: bold;
+  margin: 30px;
+}
+
+.about-paragraph {
+  font-family: 'David Libre', serif;
+  text-align: center;
+  font-size: 2.3em;
+}
+
+.about-paragraph span {
+  font-weight: bold;
+}
+
+.row {
+	display: flex;
```

Comment:
Extra tab

Reply:
I dont understand whats wrong with my about.scss. In visual studio all tabs in correct form